### PR TITLE
R95

### PR DESCRIPTION
**Pull request by chuwy**

### CODE REVIEW

**Comment by chuwy** [comment]
@alexanderdean this is done. 

Dev mirror works: http://iglucentral-dev.com.s3-website-us-east-1.amazonaws.com/release/r95/schemas/com.snowplowanalytics.snowplow/ua_parser_config/jsonschema/1-0-1

**Comment by alexanderdean** [comment]
Sorry to be a pain, can you preface the non-schema tickets with "Infrastructure: lowercase next word"? Like:

https://github.com/snowplow/iglu-central/blob/master/CHANGELOG#L511

**Comment by chuwy** [comment]
@alexanderdean done!

File: ci/lint.bash [hunk 1]
```diff
@@ -38,4 +34,5 @@ echo "========================"
 echo "LINTING ALL JSON SCHEMAS"
 echo "------------------------"
 
-java -jar ${install_dir}/${validator_jar} lint "schemas/"
+java -jar ${IGLUCTL} lint "schemas/"
```
Comment (BenFradet) [inline ci/lint.bash]: :fire:

**Pull request merged**